lib/src/request/form/from_form_value.rs

```diff
@@ -73,21 +73,11 @@ impl<'v> FromFormValue<'v> for String {
 
     // This actually parses the value according to the standard.
     fn from_form_value(v: &'v str) -> Result<Self, Self::Error> {
-        let result = URI::percent_decode(v.as_bytes());
+        let replaced = v.replace("+", " "); // keep `v` as is, so we may return it
```

Comment:
This results in two `String` allocations, where the function previously made one. Can we do this in one allocation without too much effort?